templates/layouts/_questionnaire.html

```diff
@@ -34,7 +34,7 @@
 
   {% block after_submit_button_content %}
     {% if content.return_to_hub_url %}
-      <p class="u-mt-xl"><a href="{{ content.return_to_hub_url }}">{{ _("Choose another section and return to this later") }}</a></p>
+      <p class="u-mt-xl"><a href="{{ content.return_to_hub_url }}" data-ga-action="Choose another section click">{{ _("Choose another section and return to this later") }}</a></p>
```

Comment:
This seems more like a value and prob should be like `choose-another-section-click` but 🤷